src/logic/operate.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import Big from 'big.js';
+
+export default function operate(numberOne, numberTwo, operation) {
+  const one = Big(numberOne);
+  const two = Big(numberTwo);
+  if (operation === '+') {
+    return one.plus(two).toString();
+  }
+  if (operation === '-') {
+    return one.minus(two).toString();
+  }
+  if (operation === 'x') {
+    return one.times(two).toString();
+  }
+  if (operation === '÷') {
+    return one.div(two).toString();
+  }
```

Comment:
- [ ] When I try to divide a number with `zero` like `5/0` am getting this error
![e](https://user-images.githubusercontent.com/44745855/129589241-96b00bc7-a90e-4bd5-aa84-7348132b3995.png)

It's true is an `error` but it has to be displayed on the screen you can check in your normal calculator.. please fix that

Reply:
- This issue is well resolved :+1: